fresh.py

```diff
@@ -155,6 +155,7 @@ def main():
         try:
             spotifyObj.user_playlist_remove_all_occurrences_of_tracks(user.username, user.playlist, tracks)
             results = spotifyObj.user_playlist_add_tracks(user.username, user.playlist, tracks)
+            print{("{} tracks have been added.").format(len(tracks))

```

Comment:
syntax error here. this wouldnt be a solution as it would add the number of tracks that you have scrapped which is not the same as the number of **new** tracks added